3.10-rc/buster/Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		libbluetooth-dev \
 		tk-dev \
 		uuid-dev \
-	&& rm -rf /var/lib/apt/lists/*
+	&& rm -rf /var/lib/apt/lists/* \
+	&& rm -rf /var/cache/*
```

Comment:
Don't you want to just clear `/var/cache/apt/` and not any other caches that are there?